### PR TITLE
Copies the footer from Fireplace.

### PR DESCRIPTION
**Pull request by chuckharmston**
Design, not code.

r? @ngokevin 

screen shot 2015-10-01 at 4 10 28 pm

### CODE REVIEW

File: src/media/js/site/components/footer.js [hunk 1]
```diff
@@ -6,10 +6,31 @@ export default class Footer extends React.Component {
...
+          Except where otherwise <a href="http://www.mozilla.org/about/legal.html#site">
+          noted</a>, content on this site is licensed under
+          the <a href="http://creativecommons.org/licenses/by-sa/3.0/">Creative
+          Commons Attribution Share-Alike License v3.0</a> or any
+          later version.
         </p>
+        <nav>
+          <ul>
+            <li>
+              <a href="https://marketplace-dev.allizom.org/privacy-policy">
```
Comment (ngokevin) [inline src/media/js/site/components/footer.js]: this is hardcoded to -dev, urljoin with the MKT_ROOT or w/e

**Comment by ngokevin** [comment]
r+wc

File: src/media/js/site/components/footer.js (hunk 1 above)
Comment (chuckharmston) [inline src/media/js/site/components/footer.js]: lololol whoops.

Comment (chuckharmston) [inline src/media/js/site/components/footer.js]: /g

**Pull request merged**